src/index.html

```diff
@@ -15,13 +15,392 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__inner">
+        <a
+          class="logo"
+          href="#home"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a
+                class="header__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                class="header__link"
+                href="#"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="grid">
+          <div
```

Comment:
Use the semantic tag for card
```suggestion
          <article
```